doc/source/whatsnew/v1.1.0.rst

```diff
@@ -172,6 +172,7 @@ MultiIndex
                           index=[["a", "a", "b", "b"], [1, 2, 1, 2]])
         # Rows are now ordered as the requested keys
         df.loc[(['b', 'a'], [2, 1]), :]
+- Bug in :meth:`MultiIndex.get_indexer` incorrectly handling use of pad and backfill options (:issue:`29896`)
```

Comment:
can you make a separate sub section showing the before and the new results; yes its a bug fix but the effect is non-obvious. you can use the OP example.

Reply:
done, let me know if the formatting is OK by you!